src/patternfly/components/Drawer/drawer.scss

```diff
@@ -218,10 +324,89 @@ $pf-c-drawer__panel--width-list: (25, 33, 50, 66, 75);
   }
 
   @include pf-apply-breakpoint($breakpoint, $pf-c-drawer--breakpoint-map) {
-    @each $width-value in $pf-c-drawer__panel--width-list {
-      .pf-c-drawer__panel.pf-m-width-#{$width-value}#{$breakpoint-name} {
-        --pf-c-drawer__panel--FlexBasis: #{percentage($width-value / 100)};
+    // Drawer and inline
+    .pf-c-drawer.pf-m-inline#{$breakpoint-name},
+    .pf-c-drawer.pf-m-static#{$breakpoint-name} {
+      .pf-c-drawer__content {
+        flex-shrink: 1;
+      }
+
+      .pf-c-drawer__panel {
+        --pf-c-drawer__panel--BoxShadow: transparent;
+
+        &::after {
+          --pf-c-drawer__panel--after--BackgroundColor: var(--pf-c-drawer--m-inline--m-expanded__panel--after--BackgroundColor);
+        }
       }
     }
+
+
+    // Drawer inline
+    .pf-c-drawer.pf-m-inline#{$breakpoint-name} {
+      .pf-c-drawer__content {
+        overflow-x: auto;
+      }
+
+      .pf-c-drawer__panel {
+        margin-left: calc(var(--pf-c-drawer__panel--FlexBasis) * -1);
+        transform: translateX(100%);
+      }
+
+      // Inline, expanded
+      &.pf-m-expanded .pf-c-drawer__panel {
+        margin-left: 0;
+        transform: translateX(0);
+      }
+    }
+
+    // Static
+    .pf-c-drawer.pf-m-static#{$breakpoint-name} {
+      .pf-c-drawer__panel {
+        margin-left: 0;
+        transform: translateX(0);
+      }
+
+      /* stylelint-disable max-nesting-depth */
+      &.pf-m-panel-left {
+        .pf-c-drawer__panel {
+          margin-right: 0;
+          margin-left: 0;
+          transform: translateX(0);
+
+          &::after {
+            right: 0;
+            left: auto;
+          }
+        }
+      }
+      /* stylelint-enable */
+    }
+  }
+
+  @include pf-apply-breakpoint($breakpoint, $pf-c-drawer--breakpoint-map) {
+    // Drawer inline - Panel left
+    .pf-c-drawer.pf-m-panel-left.pf-m-inline#{$breakpoint-name} {
+      .pf-c-drawer__panel {
+        --pf-c-drawer__panel--BoxShadow: transparent;
+
+        margin-right: calc(var(--pf-c-drawer__panel--FlexBasis) * -1);
+        margin-left: 0;
+        transform: translateX(-100%);
+      }
+
+      // Inline, expanded, panel left
+      // stylelint-disable selector-max-class
+      &.pf-m-expanded .pf-c-drawer__panel {
+        margin-right: 0;
+        transform: translateX(0);
+      }
+      // stylelint-enable
+    }
   }
 }
+
+// Toolbar in drawer
+.pf-c-drawer .pf-c-data-toolbar {
+  --pf-c-data-toolbar__content--PaddingRight: var(--pf-c-drawer__panel-body--PaddingRight);
```

Comment:
Why doesn't the data-toolbar have left/right padding that match the page gutter system (lg on desktop, md on mobile)? I would expect it to. This is what it looks like currently in a demo with the page component and a table.

<img width="1199" alt="Screen Shot 2020-02-07 at 5 22 51 PM" src="https://user-images.githubusercontent.com/35148959/74073081-94fe3600-49ce-11ea-8e86-40b4cbfc4d6c.png">

Reply:
Also that would allow someone to use the toolbar somewhere else in the drawer in another context without getting these styles inadvertently, since the current styling will apply to all toolbars in a drawer.

Reply:
> Why doesn't the data-toolbar have left/right padding that match the page gutter system (lg on desktop, md on mobile)?

`.pf-c-page` left/right padding doesn't match most elements within which `pf-c-toolbar` can/will be used (`.pf-c-card`, `.pf-c-tab-content`, `.pf-c-wizard`, etc) or that it applies to (`.pf-c-data-list`, `.pf-c-table`, etc). If that's the expected behavior, then all components with built in padding need to follow that pattern.

Reply:
https://github.com/patternfly/patternfly-next/issues/2695